executor/builder.go

```diff
@@ -901,13 +901,15 @@ func (b *executorBuilder) buildDelete(v *plan.Delete) Executor {
 		b.err = errors.Trace(b.err)
 		return nil
 	}
-	return &DeleteExec{
-		baseExecutor: newBaseExecutor(nil, b.ctx),
+	deleteExec := &DeleteExec{
```

Comment:
can we un-export `SelectExec`, `Tables` and `IsMultiTable` ?

Reply:
we should modify it in another pr.